hapi/hedera-protobufs/services/consensus_create_topic.proto

```diff
@@ -70,4 +71,37 @@ message ConsensusCreateTopicTransactionBody {
      * If specified, there must be an adminKey and the autoRenewAccount must sign this transaction.
      */
     AccountID autoRenewAccount = 7;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset, custom fees CANNOT be set for this topic.<br/>
+     * If not set when the topic is created, this field CANNOT be set via update.
+     * If set when the topic is created, this field CAN be changed via update.
```

Comment:
We follow the same RFC (2119) specified for the HIP process here; `CAN` is not a defined keyword.
```suggestion
     * If set when the topic is created, this field MAY be changed via update.
```

Reply:
Fixed